web_src/js/markup/asciinema.js

```diff
@@ -0,0 +1,12 @@
+export async function renderAsciinemaPlayer() {
+  const els = document.querySelectorAll('.asciinema-player-container');
+  if (!els.length) return;
+
+  const player = await import(/* webpackChunkName: "asciinema" */'asciinema-player');
+
+  for (const el of els) {
+    player.create(el.getAttribute('data-asciinema-player-src'), el, {
+      poster: 'npt:1:0:0',
```

Comment:
If I read https://www.ietf.org/rfc/rfc2326.txt

>  3.6 Normal Play Time
    Normal play time (NPT) indicates the stream absolute position
    relative to the beginning of the presentation. The timestamp consists
    of a decimal fraction. The part left of the decimal may be expressed
    in either seconds or hours, minutes, and seconds. The part right of
    the decimal point measures fractions of a second.
    The beginning of a presentation corresponds to 0.0 seconds. Negative
    values are not defined. The special constant now is defined as the
    current instant of a live event. It may be used only for live events.
    NPT is defined as in DSM-CC: "Intuitively, NPT is the clock the
    viewer associates with a program. It is often digitally displayed on
    a VCR. NPT advances normally when in normal play mode (scale = 1),
    advances at a faster rate when in fast scan forward (high positive
    scale ratio), decrements when in scan reverse (high negative scale
    ratio) and is fixed in pause mode. NPT is (logically) equivalent to
    SMPTE time codes." [5]
    npt-range = ( npt-time "-" [ npt-time ] ) | ( "-" npt-time )
    npt-time = "now" | npt-sec | npt-hhmmss
    npt-sec = 1DIGIT [ "." DIGIT ]
    npt-hhmmss = npt-hh ":" npt-mm ":" npt-ss [ "." DIGIT ]
    npt-hh = 1DIGIT ; any positive number
    npt-mm = 12DIGIT ; 0-59
    npt-ss = 12DIGIT ; 0-59
    Examples:
    npt=123.45-125
    npt=12:05:35.3-
    npt=now-

correctly, shouldn't this be `npt:0:0:0`/`npt:0.0`?
Also, we should add a comment here, we can't expect readers to know this obscure standard, and be able to interpret it correctly.
```suggestion
      poster: 'npt:0.0', // Start from the beginning, https://www.ietf.org/rfc/rfc2326.txt
```

Reply:
Actually, `poster` is not the start time, it's "preview time".

If we set it to `npt:0.0`, we will see anything in the preview before playing the "video". I tried to set it to something like `npt:end`, however, it's not supported, so I think one hour should be enough.

Reply:
I added some comments to explain it.